g_docs.go

```diff
@@ -639,6 +642,17 @@ func parserComments(comments *ast.CommentGroup, funcName, controllerName, pkgpat
 		case "OPTIONS":
 			item.Options = &opts
 		}
+
+		if len(opts.Responses) == 0 {
+			ColorLog("[WARN] missing response (@Success / @Failure) swagger doc for route `%s`\n", routerPath)
+		}
+
+		for status, resp := range opts.Responses {
+			if resp.Description == "" {
+				ColorLog("[WARN] missing description from %s Response for `%s` router\n", status, opts.Description)
```

Comment:
perhaps consider log the `routerPath` here?

Reply:
good call! but `routerPath` in here is just the inner router path, not the full one, so we get a lot of empty router path. eg.
- instead of `/customer/` it only holds the `/` part of the full router path
- instead of `/payment/midtrans` it only holds `/midtrans`.

so I decided to refactor the warnings altogether and move the code to just before we generate the swagger files. This way the information is complete and would also be easier for dev to locate the error part of the golang code. Addressed in https://github.com/zalora/bee/pull/20/commits/198c8432a0ac0a0b30ce1c918b014fe3c6c0219a